data/config-schema/src/main/resources/net/corda/schema/configuration/db/1.0/corda.db.json

```diff
@@ -64,6 +64,30 @@
                   "type": "null"
                 }
               ]
+            },
+            "idleTimeout": {
+              "description": "The maximum time (in minutes) a connection can stay idle in the pool. A value of 0 means that idle connections are never removed from the pool.",
+              "type": "integer",
+              "minimum": 0,
+              "default": 2
+            },
+            "maxLifetime": {
+              "description": "The maximum time (in minutes) a connection can stay in the pool, regardless if it is idle or not.",
```

Comment:
I don't think this description is accurate, or at least it contradicts the Hikari data source factory which states
`An in-use connection will never be retired, only when it is closed will it then be removed.`
The description here reads like you would kick a non-idle connection out of the pool, is that your intention?

Reply:
Yes you are right, good catch

Reply:
Changed to say an in-use connection that has reached the timeout will get removed from pool when it gets **idle** instead of **closed** which is what the Hikari docs mention